### ISSUE TITLE
Build failure on Arch Linux: implicit declaration of 'wlc_set_selection' did you mean ‘wlc_set_touch_cb’?

### ISSUE THREAD

**Issue by fusion809** [SirCmpwn/sway#1280]
Hi,

On Arch Linux a PKGBUILD that worked fine at building rc3 fails with rc4, giving the error:

```bash
/home/fusion809/AUR/sway/src/sway-0.14-rc4/sway/handlers.c: In function ‘handle_wlc_ready’:
/home/fusion809/AUR/sway/src/sway-0.14-rc4/sway/handlers.c:1096:2: error: implicit declaration of function ‘wlc_set_selection’; did you mean ‘wlc_set_touch_cb’? [-Werror=implicit-function-declaration]
  wlc_set_selection("test", &type, 1, &clip_test_cb);
  ^~~~~~~~~~~~~~~~~
  wlc_set_touch_cb
cc1: all warnings being treated as errors
```

My PKGBUILD is:

```bash
# $Id$

pkgname=sway
pkgver=0.14.rc4
_pkgver=${pkgver/.r/-r}
pkgrel=1
pkgdesc="i3 compatible window manager for Wayland"
arch=("i686" "x86_64")
url="http://swaywm.org"
license=("MIT")
backup=("etc/sway/security")
depends=(
	"wlc" "xorg-server-xwayland" "json-c" "pango" "wayland" "gdk-pixbuf2"
)
optdepends=(
	"rxvt-unicode: Default terminal emulator."
	"dmenu: Default for launching applications."
	"imagemagick: For taking screenshots."
	"ffmpeg: For recording screencasts."
	"i3status: To display system information with a bar."
)
makedepends=("cmake" "asciidoc")
source=(
#	"$pkgname-$pkgver.tar.gz::https://github.com/SirCmpwn/$pkgname/archive/${_comm}.tar.gz"
	"$pkgname-$pkgver.tar.gz::https://github.com/SirCmpwn/$pkgname/archive/${_pkgver}.tar.gz"
)
install="$pkgname.install"
sha256sums=('0d779df139e589cbc05a00ee3fe0804390475c8e6c59a63c0eba56c601bfe8f8')

build() {
        SRC=$srcdir/sway-${_pkgver}
        cd $SRC
	mkdir -p build
	cd build
	cmake "$SRC" \
		-DCMAKE_BUILD_TYPE=Release \
		-DCMAKE_INSTALL_SYSCONFDIR=/etc \
		-DCMAKE_INSTALL_PREFIX=/usr \
		-DVERSION="${_pkgver}"
	make
}

package() {
        SRC=$srcdir/sway-${_pkgver}
        cd $SRC
	cd build
	DESTDIR="$pkgdir" make install
	install -Dm644 "$SRC/LICENSE" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
}
```

Thanks for your time

**Comment by SirCmpwn** [comment]
Update wlc to master.

**Comment by fusion809** [comment]
Built wlc from master (using the [wlc-git](https://aur.archlinux.org/packages/wlc-git) package) and retried and got this error instead:

```bash
/home/fusion809/AUR/sway/src/sway-0.14-rc4/sway/commands/move.c: In function ‘cmd_move’:
/home/fusion809/AUR/sway/src/sway-0.14-rc4/sway/commands/move.c:165:4: error: ‘wlc_pointer_get_position’ is deprecated [-Werror=deprecated-declarations]
    wlc_pointer_get_position(&origin);
    ^~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/fusion809/AUR/sway/src/sway-0.14-rc4/sway/commands/move.c:3:0:
/usr/include/wlc/wlc.h:513:6: note: declared here
 void wlc_pointer_get_position(struct wlc_point *out_position);
      ^~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```

**Comment by SirCmpwn** [comment]
A fix for that will be available in rc5. [This PR](https://github.com/SirCmpwn/sway/pull/1272)
